adam_zadanie7/src/test/java/zadanie7/page/LoginPage.java

```diff
@@ -0,0 +1,49 @@
+package zadanie7.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+public class LoginPage extends Page {
+
+    @FindBy(xpath = "//input[@name='username']")
+    private WebElement login;
+
+    @FindBy(xpath ="//input[@name='password']")
+    private WebElement password;
+
+    @FindBy(xpath = "//input[@value='Log In']")
+    private WebElement loginButton;
+
+    @FindBy(xpath = "//a[contains(text(),'Register')]")
+    private WebElement registerButton;
+
+
+
+    public LoginPage(WebDriver driver, String url) {
+        super(driver);
+        driver.get(url);
+    }
+
+    public LoginPage setLogin(String inputLogin) {
+        login.sendKeys(inputLogin);
+        return this;
+    }
+
+    public LoginPage setPassword(String inputPassword) {
+        password.sendKeys(inputPassword);
+        return this;
+    }
+
+    public LoginPage clickLoginButton() {
+        loginButton.click();
+        return new LoginPage(driver, "http://parabank.parasoft.com/");
```

Comment:
tu wystarczy this - metoda wywolana na obiekcie zwrca jakby ten sam obiekt - return this;
Tak jak w metodzie wyzej. Nie musisz tworzyc nowego obiektu, jesli pozostajesz na tej samej stronie.